language/google/cloud/language/syntax.py

```diff
@@ -20,7 +20,45 @@
 
 
 class PartOfSpeech(object):
-    """Part of speech of a :class:`Token`."""
+    """A Google Cloud Natural Language API Part of speech object.
+    Rich data about the Part of Speech of a token returned by the API
```

Comment:
Can you add a blank line between these two. Motivation, from [PEP257][1]

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.

[1]: https://www.python.org/dev/peps/pep-0257/